Add function to register new spellings.

misrasaurabh1 opened the pull request:
This is to add new spellings to the default spellings that comes with Language Tool. This is very important when initializing the Language Tool server while deploying it in new and unique scenarios.

Feel free to extend the functionality to other files like "prohibited.txt" and "ignore.txt" etc.

jxmorris12 commented:
@misrasaurabh1 so each LT version ships with `org/languagetool/resource/en/hunspell/spelling.txt`?

misrasaurabh1 commented:
Moved the registration of spellings to server __init__. This is because once the server has initialized, it won't reload the new spellings. 
Also I think the structure of Language Tool project should not change majorly but I am not a dev of that project so I would not have insights. Anyway I added a error if this python package fails to detect that file.
